src/scrabble.js

```diff
@@ -1,5 +1,98 @@
-function scrabble() {
 
+// Creating a function called scrabble - assigning 'word' (can use any word) as a parameter(variable that the function will have access to when it is called). In function body after {} is where logic is displayed, therefore, using an if statement to that is equal to null and returns 0
+
+// According to w3schools - "\t" - meaning of this is tabs and "\n" - meaning of this is newline character - therefore, we want to return a score of "0" if tab or newline is entered.
+
+// Remove whitespace - returns 0 for white space - according to MDN, we can use the 'trim' method that removes the whitespace.
+// const greeting = '   Hello world!   ';
+// For e.g,
+// console.log(greeting);
+// Expected output: "   Hello world!   ";
+
+// console.log(greeting.trim());
+// Expected output: "Hello world!";
+
+// Creating an if statement - that if a word has a length of 0 then this should return a score of 0
+
+// If words are entered in capitals then need to assign the toUpperCase() method to string
+
+function scrabble(word) {
+  if (word === null && word === '\n' && word === '\t' && word.length === 0) {
+    return 0
+  }
+}
```

Comment:
i think this is the right idea for these special characters but a for loop needs to come after the if statement to iterate through each letter, adding together the values of each letter

Reply:
so if....
 else... for loop